tests/functional/adapter/utils/test_array_concat.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2022 Dremio Corporation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from dbt.tests.adapter.utils.test_array_concat import BaseArrayConcat
+from tests.fixtures.profiles import unique_schema, dbt_profile_data
+
+
```

Comment:
Let's add @pytest.mark.skip(reason=" ") here, instead of running the test and getting failures.

Reply:
Makes sense makes sense.